dotcom-rendering/src/static/css/light.css

```diff
@@ -0,0 +1,4 @@
+:root {
+	--white: #ffffff;
+	--black: #121212;
```

Comment:
Could this be generated programatically from source?

Reply:
I meant could we add a comment, sorry, you’ve obviously mentioned this in the PR description